.github/workflows/assign-dependabot.yml

```diff
@@ -0,0 +1,28 @@
+name: Assign Dependabot PRs
+
+on:
+  pull_request:
+
+jobs:
+  label_pr:
+    if: github.actor == 'dependabot[bot]'
+    permissions:
+      pull-requests: write
+    env:
+      GH_TOKEN: ${{ github.token }}
+      PR_ID: ${{ github.event.number }}
```

Comment:
Out of curiosity, why not using `${{ github.event.number }}` directly? It's only used once